### PR TITLE
Limits the number of CPU usage for the BPR according to user parameters

### PR DESCRIPTION
**Pull request by pedrocamargo**

### CODE REVIEW

**Comment by pedrocamargo** [comment]
An easy one, @janzill

File: aequilibrae/paths/bpr.pyx [hunk 1]
```diff
@@ -1,27 +1,31 @@
 from libc.math cimport pow
 from cython.parallel import prange
 
-def bpr(congested_times, link_flows, capacity, fftime, alpha, beta):
+def bpr(congested_times, link_flows, capacity, fftime, alpha, beta, cores):
+    cdef int c
```
Comment (janzill) [inline aequilibrae/paths/bpr.pyx:5]: Is there a reason you declare and initialise in different places?

**Comment by janzill** [comment]
True, but I still have a question :D

File: aequilibrae/paths/bpr.pyx (hunk 1 above)
Comment (pedrocamargo) [reply aequilibrae/paths/bpr.pyx]: Yes.  My poor programming skills. Solved now (and incorporated elsewhere)

**Review by janzill — APPROVED** [review]

**Comment by janzill** [comment]
merge when passing tests

**Pull request merged**